### PR TITLE
Explain how `list` is useful in UG

### LINKED ISSUE #96: Update UG to explain how `list` is useful

### PR DESCRIPTION
**Pull request by LargeCrowd**
closes #96

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -72,9 +72,9 @@ Shows a message listing out all the available commands and their purpose.
 
 Format: `help`
 
-For more information regarding the comamnd formats and examples, press `F1` to open up a help window (as shown in the picture below).
+For more information regarding the command formats and examples, press `F1` to open up a help window (as shown in the picture below).
 To close the help window, you can simply press `esc` on your keyboard (other methods such as `alt`+`F4` and clicking on
-the red `X` button on the top right corner of the window also works).
+the red `X` button in the top right corner of the window also works).
```
Comment (E0735389) [inline docs/UserGuide.md:77]: Not issue with this pull request, but in the meantime can you clarify that these methods (alt+F4 and red X) **only apply for Windows**

File: docs/UserGuide.md [hunk 2]
```diff
@@ -108,6 +108,9 @@ Shows a list of all persons in the address book.
 
 Format: `list`
 
+* After using commands like `find`, the list may no longer display all contacts.
+* Use `list` to revert the list to the full list of contacts in the order they were added.
```
Comment (E0735389) [inline docs/UserGuide.md:112]: Looks good but it's probably better to put this under `find` command, and here just say "see `find` command's documentation (⟵ link to corresponding section) to know in which case this command is useful"

**Review by E0735389 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (LargeCrowd) [reply docs/UserGuide.md]: Can check this and the below formatting as well

**Review by E0735389 — APPROVED** [review]
Okay, but can you resolve the merge conflict quickly?

**Pull request merged**